### ISSUE TITLE
Advertisment in @react-spring/core/package.json breaks yarn 3

### ISSUE THREAD

**Issue by elderapo** [pmndrs/react-spring#1668]
## 🐛 Bug Report

Basically, what's written in the title... [Advertisment](https://github.com/pmndrs/react-spring/blob/master/packages/core/package.json#L30) in @react-spring/core/package.json breaks yarn 3. Simple yarn install breaks with the following error:
```
# This file contains the result of Yarn building a package (@react-spring/core@npm:9.0.0-rc.3)
# Script name: postinstall

[eval]:1
console.log('ESC[35mESC[1mEnjoy react-spring? You can now donate to our open collective:ESC[22mESC[39m
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: Invalid or unexpected token
    at new Script (vm.js:100:7)
    at createScript (vm.js:261:10)
    at Object.runInThisContext (vm.js:309:10)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1075:30)
    at evalScript (internal/process/execution.js:92:25)
    at internal/main/eval_string.js:23:3

```

## Expected behavior

No error.

## Environment

- `react-spring` v9.2.4
- `react` doesn't matter

**Comment by elderapo** [comment]
Also created [a relevant issue](https://github.com/yarnpkg/berry/issues/3361) in the yarn repo.

**Comment by JasonKleban** [comment]
Always good when a solicitation for donations breaks the users' whole environment.

**Comment by flying-sheep** [comment]
Both. I fixed the react-spring issue in #1740

**Comment by chukanovdk** [comment]
You can also use 
`  "dependenciesMeta": {
    "@react-spring/core": {
      "built": false
    }
  },`
  for yarn 3
  
  https://yarnpkg.com/configuration/yarnrc#enableScripts
  https://yarnpkg.com/configuration/manifest#dependenciesMeta
